### PR TITLE
NEWS: Updating news file before RC2

### PR DESCRIPTION
**Pull request by shamisp**
* Updating news file
* Copyright dates were bumped up

Once it is merged we tag RC2

### CODE REVIEW

**Comment by shamisp** [comment]
Ok to test

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 23-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/1c569dc07cf45ed1f40178e706bd42825b84f2ca/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/72fcf75eba725600091c0622a6d9fe48fa5004f5/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/1cd004ba510fe5e48e1baf49db36540cc1f4c580/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/3d653a1037f6e3e3c13dd3ed1aa760644e04d909/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/cfbd6c96fcf723618f7f611e20ecef22b2b57cf0/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/627c6385f29d46dc81defd867cd7acd7d9029eed/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/5ddb0d8fbc3768f213be855853b6c8e4ac0ea447/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/039e9616a3c44332b79386abad854c1013825d8f/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/52b1a18d7dd38d3a4d3e18eb2209dc7968626e74/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/01a6d5c32c916c7575ad92b5a78138d5c7e0b491/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/2dd50199aa73fd347fb33dc9703279c3564ebabf/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/ce63103c3e9fbb0cb50883f6b012cad303d6e8c4/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/bb2bbbe2efa96b8f6a332399c54996e6f94d0421/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/8592144f9b973e9b60984b3816f1fa7885d8c271/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/6c41c7d41ff57b38a66fd89a235512b4f5e8233b/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/759da275fa49c2fcf287ad681a103b3359f19cbb/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/a2077f44ccc835dc71c5dc348809adc8eedd21a3/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/3a1f97a3902dd67f907981f2455d96e87834fdc5/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/f0bfc785e93765f646bceea4dec6f9887e7b6f23/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/b9cd1e553856057fd95d3d029dc50a6fa8ab5b20/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/9f9ce8a41fa084145cf8bc4295eb5eb1b6fbc59d/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/37b904374c62b61d617524b5762dc3c2074aeb19/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/6cf19f4b5f76e6f4437e279f4b34d7bcd2be6352/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/b42bdd3816b3a1da4f5dbd8cdff0b27baaac976c/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/64e387611744f5a7aab39a52d03a1990/raw/dc22120aa5a77d5285d05087e797f75ef751c11d/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 23-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/ab64f0687e17ad26266049ab0998513ca0c6dbba/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/f6f81ca08383e52cfaee7285cf6ffbdedbde7fa6/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/96708dd4dc127688abbb413f09dea8f00cadeb03/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/309a04cc54d8fc2b257ed7795f452c0a59bd118f/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/56fd82a0b959d2daca1c5c658a92cfe92242d776/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/fc494ff09a64ecdc1be32f8bf788cc0c724e19e9/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/8e3e9fd0edb9400a601ef93db3d6583d6a28f0e6/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/00999c89e3bd74557ce5410213dd0ca43e475d6e/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/55d0134a506989cc1dee9912fe6368c73cad8391/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/05eef77b5df8ff0c5bf622f3856932ede4873217/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/20aa24de6834d0824e62730f43fd1f01e1c086b2/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/44914daa8ea5ecc9b644ef7fc5ca376734766b7a/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/0884e095e0c12cd542fff274ba995bb87956f22b/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/35f782ec34f3a6fdb2931c9c9fc2589e236dd3ea/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/1bc101feeec9627211d4f5fffccf8b01d08976bb/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/934df8249ecb85ea45dd45ddc290835e83847597/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/865013d9144693d609430c8b18a32939359a4969/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/3dd8c6cd8ef5f90880d564541832e44669a7dec1/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/258cb63b975bfc90e0d59d2b0ae4362c5ec664c5/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/09c757141018cbf19c9a2dbcfb6dc78e994fc204/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/951fd9a5a34fa0c40c8b8a36ce5c202ba2ed6929/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/212293f6fd790f8fcf42dbfe102daedef9cd7c79/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/998737e3f0f8fb0a07bcd9d9822faecc03cd70f7/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/46114821b333e14cc4878c3bd4d4c4d40a28a43a/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d164db85249d4c6e104673541ff1ad60/raw/4216850456831f37555595d28547e20381467c42/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: NEWS [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+## Copyright (C) ARM Ltd. 2017-2020.  ALL RIGHTS RESERVED.
 ##
 ## See file LICENSE for terms.
 ##
 #
 
 ## Current
 
-## 1.7.0-rc1 (October 28, 2019)
+## 1.7.0-rc2 (January 16, 2019)
```
Comment (brminich) [inline NEWS]: 2020?

File: NEWS [hunk 2]
```diff
@@ -24,13 +24,29 @@ Features:
 - Updated support for Mellanox DevX API
 - Added multiple UCT/TCP transport performance optimizations
 - Optimized memcpy() for Intel platforms
+- Adding protection from non-UCX socket based app connections
```
Comment (brminich) [inline NEWS]: maybe added to be consistent with other points?

File: NEWS [hunk 3]
```diff
@@ -24,13 +24,29 @@ Features:
...
+- Improved search time for PKEY object
+- Enable gtest over IPv6 interfaces
+- Updated Mellanox and Bull device IDs
+- Added support for CUDA_VISIBLE_DEVICES
+- Increasing limits for CUDA IPC registration
```
Comment (brminich) [inline NEWS]: Increased?

File: NEWS (hunk 1 above)
Comment (shamisp) [reply NEWS]: I didn't finish building my Time Machine, so has to be fixed

File: NEWS (hunk 2 above)
Comment (shamisp) [reply NEWS]: valid point. I will fix

**Comment by shamisp** [comment]
@brminich - Addressed. I'm not going to wait for full CI cycle on this change, since it is not functional. If you see anything else please let me know. Otherwise I will go ahead and tag RC2.

**Comment by shamisp** [comment]
@brminich thanks !

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 5 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 23-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/ae45088190b64b8a05d43645d7e32eb224d4b999/_main) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/0dde72218fb3d9a29082f3e599c7317061cf598f/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/e8037a19fd6a77868c959227ef830701466c03f8/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/821f892a5b1ed298cbacfbb3a5338d7d74e723be/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/a88fccb0e1ee9881791de1f8c8e606f3fefa14b2/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/68e6545c33c8eb67d78a1369743e3c48a385b315/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/0e9a7daf23aebbe42efef13bc3c9390574de820d/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/185acb5475c945784c5f90c4108c2510797906a1/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/2bc7c3383c045b5906b1f3ef3cfaa0b8c5f4c9a0/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/9a832a87e8b6d9bed481d2b85de67df8def0d113/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/4957c084ea4fd449bece5433f5bddabd7ef46b05/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/7e3112eba0de2f99694deec57521b353c7b8238b/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/8d4a24f87400dbf5e3b86bdb2d51db97ea531c07/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/1f58fd4ea8a45d7529289b8733ed3875d9bc8c77/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/ba7d0d3139820141bd8f2091abc7fdebf26217e6/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/953c2ccb8251373afe4c06aab25200244842160f/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/7f255e7c05010d9a4afaddefd9c005d3b87b35f9/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/f919d37ff7b7523b2e4e93b0deb6c2b2d0466741/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/f227fa13c5653095e0adf8de0c490c4b96bccbdf/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/48e9010e73701f3224d0ddb40bc7bdcd15768556/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/b201c46bea3e5785a6c044b5bbb816e3c8520b0b/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/cf79ef4dda6a77381540288f04b93086f76dbb74/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/edc8eb78241db4663ecb35287e86b74e13f8bbb5/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/c744e39a51686ba78a0d2660d06c81580c8e8a2b/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5c99105c64895c581022a76723322dee/raw/f66f7dd54ad875014c0b031db6105019b7239e97/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**